shared-actions/npm-setup/action.yml

```diff
@@ -0,0 +1,35 @@
+name: JS/TS environment setup
+description: 'Set up node and npm config, get repo and install its dependencies'
+inputs:
+  gh-token:
+    description: 'Github Token'
+    required: true
+  cache:
+    description: 'Use cache'
+    default: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
```

Comment:
`uses` is not supported yet in composite actions steps, only `run` 😞 
https://github.com/actions/runner/issues/646
I'll have to take this one out